### DISCUSSION TITLE
There is a way to auto generate my APIs documentation? (Like Swagger)

### DISCUSSION THREAD

**Discussion by wilsonneto-swe** [vercel/next.js#13743] [Help]
Hi guys, 
I already used some libs to document some apis with express and swagger, there is some ways to auto generate a api documentation, using the '/pages/api" folder?

Thanks so much!

**Comment by ImranAhmed** [comment]
Did anyone find a solution for this? Thanks. Would love to see something like Swagger for NextJS.
